include/dpp/guild.h

```diff
@@ -203,6 +203,8 @@ class DPP_EXPORT guild_member {
 	time_t joined_at;
 	/** Boosting since */
 	time_t premium_since;
+	/** total permissions of the member in the channel, including overwrites, returned when in the interaction object */
+	permission permissions;
```

Comment:
the permissions are stored in the interaction event itself to reduce storage as they're mostly empty because they're only retrieved through interaction events iirc

Reply:
not writing `interaction.resolved.member_permissions.find(interaction.usr.id)->second->has(dpp::p_ban_members)`, sorry

Reply:
this would inflate the guild records (remember there are potentially hundreds of thousands of these cached in ram) for a value that is only used in an interaction. as commandserver said get_resolved_permissions is the way to do this, the permissions are per user not per guild.